### ISSUE TITLE
Sign in with Apple Button

### ISSUE THREAD

**Issue by zbabtkis** [MichalSzorad/react-social-login-buttons#41]
Are there any plans to add a Sign in with Apple button? Seems especially useful for web based iOS applications that are now required to include Sign in with Apple. I'd be happy to submit a PR.

**Comment by CRivlaldo** [comment]
You can implement it with the `createButton()` function.

Here is my approach:

```
import React from "react";
import { createButton } from "react-social-login-buttons";
import AppleIcon from "./apple.svg";

const config = {
  text: "Sign in with Apple",
  icon: (props) => <AppleIcon {...props} style={{ fill: "white", width: "26px", height: "26px" }} />,
  style: { background: "black" },
  activeStyle: { background: "#293e69" }
};

const SignInWithAppleButton = createButton(config);

export default SignInWithAppleButton;
```

In this case, you need to find the Apple icon.

**Comment by MichalSzorad** [comment]
Thank you for the request. This should be now available in the version 3.3.0.
